src/common/logging.cpp

```diff
@@ -77,8 +78,14 @@ void otbrLogSetLevel(otbrLogLevel aLevel)
     sLevel = aLevel;
 }
 
+/** Enable/disable logging with syslog */
+void otbrLogDisableSyslog(bool aDisabled)
```

Comment:
This function is probably hard for code viewers to understand because it's the opposite direction of enabling according the flag. Probably you can rename it as `otbrLogSyslogSetEnabled` instead.